packages/core/src/components/tag/tag.tsx

```diff
@@ -26,7 +26,7 @@ export interface ITagProps extends IProps, IIntentProps, React.HTMLAttributes<Ta
      * Click handler for remove button.
      * Button will only be rendered if this prop is defined.
```

Comment:
sure, I'll rename to `tagProps`, at which point I think it's fairly clear what props are being passed, but happy to also add to the description if you'd like